horace_core/sqw/PixelData/@PixelData/get_pixels.m

```diff
@@ -0,0 +1,101 @@
+function pix_out = get_pixels(obj, abs_pix_indices)
+% GET_PIXELS Retrieve the pixels at the given indices in the current page, return
+% a new PixelData object
+%
+%  >> pix_out = pix.get_pixels(1:100)  % retrieve pixels at indices 1 to 100
+%
+%  >> pix_out = pix.get_pixels([1, 0, 1])  % retrieve pixels at indices 1 and 3
+%
+% The function attempts to mimic the behaviour you would see when indexing into
+% a Matlab array. The difference being the returned object is a PixelData
+% object and not an array.
+%
+% This function may be useful if you want to extract data for a particular
+% image bin.
+%
+% Input:
+% ------
+%   abs_pix_indices  A vector of positive inegers or a vector of logicals.
+%                    If a vector of integers, include the pixels with those
+%                    indices, in the given order, in the returned PixelData
+%                    object.
+%                    If a vector of logicals, keep pixels where the logical
+%                    array index is 1 and remove pixels where it's 0.
+%
+% Output:
+% -------
+%   pix_out        Another PixelData object containing only the pixels
+%                  specified in the abs_pix_indices argument.
+%                  This PixelData object will hold all its pixels in memory and
```

Comment:
This is only true as a consequence of the default constructor and will be changing.

Unless this information is absolutely critical here I suggest removing it as it will be incorrect when a change is made to another piece of code.